python/pyspark/mllib/recommendation.py

```diff
@@ -105,9 +112,15 @@ class MatrixFactorizationModel(JavaModelWrapper, JavaSaveable, JavaLoader):
     ...     pass
     """
     def predict(self, user, product):
+        """
+        Predicts rating for a given user and product.
+        """
         return self._java_model.predict(int(user), int(product))
 
     def predictAll(self, user_product):
+        """
+        Returns a list of predicted Ratings for a user and many products.
```

Comment:
`predicted ratings for input user and product pairs`. This works for more than one user.